gcloud/datastore/demo/demo.py

```diff
@@ -1,16 +1,3 @@
-# Copyright 2014 Google Inc. All rights reserved.
```

Comment:
I'm not sure about the legal aspects of this... 

Maybe we can use 2 ##'s and skip lines that start with those in the runner script?